### ISSUE TITLE
FR: Improve "Retry Failed Files Only" Workflow

### ISSUE THREAD

**Issue by pinuke** [brianpetro/obsidian-smart-connections#79]
## Support Keyboard Navigation and Add "Retry Failed Files Only" to the Command Palette

![image](https://user-images.githubusercontent.com/12621101/227327419-0481363b-a1b1-4cbf-8a39-56620599416e.png)

It would be nice to be able to set a my own keyboard shortcut to run "Retry Failed Files"/have it show up in the menu for Ctrl+P

Since retry failed files is something that I frequently have to do while testing for bugs, it would be nice to have a way to do this faster.

## Add a "Retry Failed Files Continuousy Until They're Cleared Out" Command/Toggle

Since, most Failed File Embeds are caused by Error Code 429/rate limits for the minute, it would make sense to implement a command that retries the failed files after 60 or more seconds, since your rate limit should have reset.

For this, I recommend imposing a Retry limit. After 3 failed retries, revert to the old Retry Failed Files functionality.

**Comment by brianpetro** [comment]
Great suggestions, thanks, @smartguy1196 !

**Comment by brianpetro** [comment]
Closing because no existing UI for failed files since v2 🌴

**Comment by brianpetro** [comment]
recorded response https://youtu.be/Pggi1WFdS6s
